scripts/openshift-helpers.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, 2022 Red Hat, IBM Corporation and others.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+###############################  v OPENSHIFT v #################################
+
+function check_prometheus_installation() {
+	echo
+	echo "Info: Checking pre requisites for openshift..."
+	kubectl_tool=$(which kubectl)
+	check_err "Error: Please install the kubectl tool"
+	# Check to see if kubectl supports kustomize
+	kubectl kustomize --help >/dev/null 2>/dev/null
+	check_err "Error: Please install a newer version of kubectl tool that supports the kustomize option (>=v1.12)"
+
+	kubectl_cmd="kubectl"
+	prometheus_pod_running=$(${kubectl_cmd} get pods --all-namespaces | grep "prometheus-k8s")
+	if [ "${prometheus_pod_running}" == "" ]; then
+		echo "Prometheus is not running, use 'scripts/prometheus_on_minikube.sh' to install."
```

Comment:
This error message is only for minikube. For Openshift we need to print "Either Prometheus is not running or it is not accessible. Please check permissions and try again"

Reply:
Done